.github/workflows/python-app.yml

```diff
@@ -51,7 +50,7 @@ jobs:
 
       - name: Run pylint
         run: |
-          pylint --load-plugins pylint_django tcf_website tcf_core
+          pylint --disable=R0801 --load-plugins pylint_django tcf_website tcf_core

```

Comment:
Would it be better to individually have `pylint disable` lines in the specific files where this is an issue? I think there's also a way to have pylint config files, but not sure if that's worth the extra effort.

Reply:
I did this because it's a cross-file check, but now I don't see the error because of the changes I made afterwards, so I removed the flag entirely in d8436a63dc705ee2b4aa07444dfb3adc975dacea.